src/integrationtest/python/should_exclude_optional_tasks.py

```diff
@@ -0,0 +1,47 @@
+#   -*- coding: utf-8 -*-
+#
+#   This file is part of PyBuilder
+#
+#   Copyright 2011-2015 PyBuilder Team
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import unittest
+
+from integrationtest_support import IntegrationTestSupport
+
+
+class Test(IntegrationTestSupport):
+    def test(self):
+        self.write_build_file("""
+from pybuilder.core import init
+
+@task
+def a(project):
+    project.set_property("a", True)
+
+@task
+@depends(optional=("a"))
+def b(project):
+    project.set_property("b", True)
+        """)
+        reactor = self.prepare_reactor()
+        project = reactor.project
+
+        reactor.build("b")
```

Comment:
Why is the optional dependency on `a` not pulled in? I thought this behaviour would require me to `-x a`